### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by ivan-strelka** [Shardeum/shardeum-bug-reporting#465]
##### What is the issue?

:red_circle:  Describe the error in simple terms. :red_circle: 
    
#### What impact does the issue have?

:red_circle: Describe if the bug is a visual warning or if it breaks functionality causing a system to fail. :red_circle: 

##### How to reproduce the issue? 

:red_circle: 
Explain in as much detail as possible to help reproduce the bug:
```
    -where the error is seen
    -how the error was potentially caused if possible (mention previous steps taken to see the error if relevant)
    -share code and explain how the code was run
```
:red_circle: 

##### What other resources can you share regarding this issue?

:red_circle: Photos and links related to the error. :red_circle: 

###########################
# 5. Start Compose Project #
############################

Creating shardeum-dashboard ... done
Starting image. This could take a while...
npm ERR! code ENOENT
npm ERR! syscall mkdir
npm ERR! path /home/node/.npm
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, mkdir '/home/node/.npm'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Log files were not written due to an error writing to the directory: /home/node/.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
fatal: could not create work tree dir 'cli': No space left on device
./entrypoint.sh: line 30: cd: cli: No such file or directory
fatal: could not create work tree dir 'gui': No space left on device
./entrypoint.sh: line 37: cd: gui: No such file or directory
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /home/node/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/home/node/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Log files were not written due to an error writing to the directory: /home/node/.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
...+.........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.+...+.....+.......+........+.......+......+..+.+..+....+..............+.+.........+.........+.....+.......+.........+..+.......+...+.....+.........+...+......+......+.+.....+...+.......+...............+...+..+.......+......+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..........+.....+.........+.+.....+.+...+......+......+.....+...+.......+...+........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
...+..........+.....+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.........+.+..+.+..+............+.+.........+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+...+..+......+......+.+.........+.....+.+..+.+..............+.......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
..+..................+..+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*......+.........+...........+.+..................+.....+.+......+.....+...+.+.........+.....+...+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.........+......+.....+......+.......+........+.......+..+.+...+....................+.+........+......+......+...+.......+......+..+...+..................+.+.....+......+...............+....+........+..........+...........+.+...+..+....+.....+.......+...+..+...+...................+...+.....+..........+..............+.........+.+.........+.....+..........+...........+...+...+............+.......+.....+.............+..+....+.........+......+..+.............+.........+...+..+.........................+..+.+..+......+....+...........+......+.+..+...+.........+...+...+...+....+...+...+..+.........+................+.........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
.+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..+.+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
Certificate request self-signature ok
subject=C = XX, ST = Localzone, L = Localhost, O = Shardeum Sphinx 1.x Validator Cert., CN = localhost
./entrypoint.sh: line 112: operator-cli: command not found
./entrypoint.sh: line 114: operator-cli: command not found
./entrypoint.sh: line 116: operator-cli: command not found
Reusing old node
New container id is : 0be1c040bfcf
Error: No such container:path: 0be1c040bfcf:/home/node/app/cli/build
Trying again with sudo...
Error: No such container:path: 0be1c040bfcf:/home/node/app/cli/build

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.

**Comment by cresdent1** [comment]
How do you solve this problem? I have the same problem but the error for npm commands are "illegal instruction (core dumped)" during the installation, then after "operator-cli" is not found
